python/tests/fixtures/create_test_data.py

```diff
@@ -11,18 +11,18 @@
 
 def random_vectors(rows: int, dimensions: int, dtype, seed: int = 12345) -> np.ndarray:
     rng = np.random.default_rng(seed)
-    if dtype == np.single:
+    if dtype == np.float32:
         vectors = rng.random((rows, dimensions), dtype=dtype)
-    elif dtype == np.ubyte:
+    elif dtype == np.uint8:
         vectors = rng.integers(
             low=0, high=256, size=(rows, dimensions), dtype=dtype
         )  # low is inclusive, high is exclusive
-    elif dtype == np.byte:
+    elif dtype == np.int8:
         vectors = rng.integers(
             low=-128, high=128, size=(rows, dimensions), dtype=dtype
         )  # low is inclusive, high is exclusive
     else:
-        raise RuntimeError("Only np.single, np.byte, and np.ubyte are supported")
+        raise RuntimeError("Only np.float32, np.int8, and np.uint8 are supported")
```

Comment:
I have always been bothered by scoping in Python .... every other language I use would not allow you to define "vectors" inside an if block and return it outside the if block.  I just forget about that rule.